hazelcast/src/main/java/com/hazelcast/jet/impl/execution/TaskletExecutionException.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.jet.impl.execution;
```

Comment:
```suggestion
/*
 * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package com.hazelcast.jet.impl.execution;
```

Reply:
Also maybe should go into `com.hazelcast.jet.impl.exception` package?

Reply:
> Also maybe should go into `com.hazelcast.jet.impl.exception` package?

I have mixed feeling about this. What's the reason for this package? IMHO I think we should keep exceptions close to usage, this way we can make it package-private